src/Microsoft.SqlTools.ServiceLayer/Scripting/SqlObjectIdentifier.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
It looks like this header is expected follow a specific format and I'm getting the below build error.

```
.\sqltoolsservice\src\Microsoft.SqlTools.ServiceLayer\Scripting\SqlObjectIdentifier.cs(1,1): er
ror IDE0073: A source file contains a header that does not match the required text (https://learn.microsoft.com/dotnet/
fundamentals/code-analysis/style-rules/ide0073) [.\sqltoolsservice\src\Microsoft.SqlTools.Servi
ceLayer\Microsoft.SqlTools.ServiceLayer.csproj]
```

this is the expected format.

```
//
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
//
```

Reply:
fixed